nkms/network/protocols.py

```diff
@@ -59,14 +68,19 @@ def rpc_store(self, sender, nodeid, key, value):
         self.welcomeIfNewNode(source)
         self.log.debug("got a store request from %s" % str(sender))
 
-        if value.startswith(b"uaddr") or value.startswith(b"trmap"):
-            signature, sender_pubkey_sig, hrac, message = dht_value_splitter(value[5::], return_remainder=True)
+        if value.startswith(BYTESTRING_IS_URSULA_IFACE_INFO) or value.startswith(
+                BYTESTRING_IS_TREASURE_MAP):
+            header, signature, sender_pubkey_sig, hrac, message = dht_value_splitter(
```

Comment:
In the wise words of @jMyles, "Should this be a class?"

:)

Reply:
✔️